### ISSUE TITLE
Combine LSP and MPII

### ISSUE THREAD

**Issue by MetaDev** [YuliangXiu/MobilePose-pytorch#18]
Have you tried combining both dataset LSP and MPII?
LSP has the joint "neck" and MSPII "upper_neck" and "thorax".
from the images it seems neck and upper_neck are semantically the same but I'm not sure.
Do you have any insight in this?

**Comment by YuliangXiu** [comment]
I did not combine these two datasets, but it is a valuable attempt to boost estimation accuracy further. I think you can firstly train the network only with the common joints and then finetune the final FC layer to any joint format you want.

**Comment by MetaDev** [comment]
Thanks for the advice!

**Comment by YuliangXiu** [comment]
Yes, I use regression in this project.
